src/Microsoft.Azure.WebJobs.Storage/Blob/IStorageBlobProperties.cs

```diff
@@ -37,5 +37,10 @@ internal interface IStorageBlobProperties
         /// Gets the LeaseStatus of the blob.
         /// </summary>
         LeaseStatus LeaseStatus { get; }
+
+        /// <summary>
+        /// Gets the underlying SDK object.
+        /// </summary>
+        BlobProperties SdkObject { get; }
```

Comment:
Not a fan of this naming, since it's basically a poco object I'd rather see it named as BlobProperties.  Basically passing 'sdkobject' to a script language reads weirdly to me.